security-proxy/src/main/webapp/WEB-INF/applicationContext-security.xml

```diff
@@ -14,6 +14,19 @@
     <context:property-placeholder location="file:${georchestra.datadir}/default.properties, file:${georchestra.datadir}/security-proxy/security-proxy.properties"
         ignore-resource-not-found="true" ignore-unresolvable="true" />
 
+	<bean id="allowSemicolonHttpFirewall" class="org.springframework.security.web.firewall.StrictHttpFirewall">
+		<!-- 
+			Set allowSemicolon = true to avoid errors 
+		    like 'the request was rejected because the URL contained a potentially malicious String ";"'
+		    See <https://jira.camptocamp.com/browse/GSHDF-95> 
+		    ";" in URL's come in the form http://<domain>/<path>;jsessionid=xxxx
+		    In general, proxied applications are discouraged to disclose jsessionid this way, and shall use a cookie instead.
+		    Some applications like CKAN though can't be modified to use cookies, hence this configuration.
+		 --> 
+	  	<property name="allowSemicolon" value="true"/> 
```

Comment:
I would find it useful to have this setting configurable from the datadir, with a default value set to `false`.
Platforms using CKAN would have to switch it to true.

Reply:
sounds doable, let me see

Reply:
See 26cfc9a6b064828a8eba1dca4fdc54727e741aff and https://github.com/georchestra/datadir/pull/162